src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx

```diff
@@ -334,6 +343,34 @@ const ActionsPageEvent = ({
                   </span>
                 ),
                 storageSlot: values?.slot?.toHexString(),
+                amountTag: (
+                  <Tag
+                    appearance={{
+                      theme: 'primary',
+                      colorSchema: 'inverted',
+                      fontSize: 'tiny',
+                    }}
+                  >
+                    <Numeral
+                      value={values?.stakeAmount || 0}
```

Comment:
This does not properly calculate the value for the stakeAmount.

![Screenshot from 2021-04-16 15-36-19](https://user-images.githubusercontent.com/1193222/115026428-2585af80-9ecb-11eb-8ff9-f6fc182aeb06.png)

Please revise your logic in the resolver

For reference the max stake is 995 and the amount I staked is 800

The values are in WEI

Reply:
Tested it again (Both with the staking widget and with your method) after rebasing and doing the other changes requested, and it seems to be working as expected!